manageiq-providers-vmware.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
-  s.add_dependency("fog-vcloud-director", ["~> 0.2.2"])
+  s.add_dependency("fog-vcloud-director", ["~> 0.3.0"])
   s.add_dependency "fog-core",                "~>1.40"
```

Comment:
Can we drop the extra fog-core dependency?  That way it will evaluate to whatever the `fog-*` gems need?

Reply:
Good one, didn't realize we're enforcing it from here as well, thanks.